doc/api/esm.md

```diff
@@ -538,6 +538,43 @@ and in a CommonJS one. For example, this code will also work:
 const { something } = require('a-package/foo'); // Loads from ./foo.js.
 ```
 
+### Internal package imports
+
+In addition to the `"exports"` field it is possible to define internal package
+import maps that only apply to import specifiers from within the package itself.
+
+Entries in the imports field must always start with `#` to ensure they are
+clearly disambiguated from package specifiers.
+
+For example, the imports field can be used to gain the benefits of conditional
+exports for internal modules:
+
+```json
+// package.json
+{
+  "imports": {
+    "#dep": {
+      "node": "dep-node-native",
+      "default": "./dep-polyfill.js"
+    }
+  },
+  "dependencies": {
+    "dep-node-native": "1.0.0"
+  }
+}
+```
+
+where `import '#dep'` would now get the resolution of the external package
```

Comment:
```suggestion
Where `import '#dep'` would now get the resolution of the external package
```

I assume it wasn't capitalized because the code sample above was treated as the beginning of the sentence..? I think it's less confusing capitalized. Not 100% sure what's correct though.

Reply:
In my dissertations we used to always treat equations as part of a sentence. I generally carry that writing style over to small code samples, but am open to the change too.

Reply:
Interesting! I'm too lazy to look it up and I can totally see the arguments either way. So - resolving. :)